### PR TITLE
MR-255: Routes for superadmin and member

### PR DESCRIPTION
**Pull request by maurojjzz**
created two new files on Routes folder which each one defines its routes redirection and what to render to each different user (SuperAdmin or Member in this case)

### CODE REVIEW

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

File: src/routes/superAdmin.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    name: 'Admin',
+    path: '/superAdmin/admin'
+  },
+  {
+    name: 'Log Out',
+    path: '/'
+  }
+];
+
+const superAdmin = () => {
```
Comment (FrancoLelli) [inline src/routes/superAdmin.js:19]:
```suggestion
const SuperAdminRoutes = () => {
```

File: src/routes/superAdmin.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <Switch>
+        <Route path={`${url}/superAdmin/admin`} component={Admin} />
+        <Route path={`${url}/superAdmin/admin/form/:id?`} component={FormAdmin} />
+        <Redirect path={`${url}/login`} />
+      </Switch>
+    </Layout>
+  );
+};
+
+export default superAdmin;
```
Comment (FrancoLelli) [inline src/routes/superAdmin.js:32]:
```suggestion
export default SuperAdminRoutes ;
```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]

**Review by FrancoLelli — APPROVED** [review]

**Comment by maurojjzz** [comment]
[MEMBER-SUPERADMINS ROUTE](https://trello.com/c/qnZpmLiC/255-member-superadmins-route)

**Pull request merged**